playground/terraform/build.gradle.kts

```diff
@@ -392,13 +392,17 @@ tasks.register("takeConfig") {
    var ipaddrname = ""
    var d_tag = ""
    var dns_name = ""
+   var dstore_name = ""
```

Comment:
Should we spell out the full `datastore_name`?

Reply:
Variable name was fixed